lib/game.rb

```diff
@@ -0,0 +1,88 @@
+require_relative 'board.rb'
+class Game
+  WIN_CONDITION = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [6, 4, 2]
+  ].freeze
+
+  obj_board = Board.new
+  # board = obj_board.get_board
+  obj_board.board
+
+  # define the turn - checks what happens in each turn
+  def turn(user_input, board, obj_board)
+    index = obj_board.input_to_index(user_input)
+    # rubocop: disable Style/GuardClause
+    if obj_board.valid_move?(board, index)
+      obj_board.move(board, index, current_player(board))
+    else
+      return ['Not a valid move!!! Try again']
+    end
+    # rubocop: disable Style/RedundantReturn
+    return board
+    # rubocop: enable Style/RedundantReturn
+    # rubocop: enable Style/GuardClause
```

Comment:
- [ ] Don't overuse disable rubocop comments.
It's fairly easy and good practice to follow the ruby language style, such as avoiding redundant returns and using guard clauses. Read up on the error you got before disabling them and refactor your code to follow the ruby style best practices.